minisom.py

```diff
@@ -71,8 +71,14 @@ def fast_norm(x):
     return sqrt(dot(x, x.T))
 
 
-def asymptotic_decay(learning_rate, t, max_iter):
-    """Decay function of the learning process.
+def learning_rate_inverse_time_decay(learning_rate, t, max_iter):
+    """Decay function of the learning process that asymptotically
+    approaches zero.
+
+    This function should NOT be used for the sigma_decay_function
```

Comment:
If these is not supposed to be used for sigma, it's better to have a close selection just as the neighborhood function. This how the neighborhood function is selected: https://github.com/JustGlowing/minisom/blob/master/minisom.py#L210